ads/_config.js

```diff
@@ -485,7 +485,12 @@ export const adConfig = {
   'ibillboard': {},
 
   'idealmedia': {
-    renderStartImplemented: false,
+    renderStartImplemented: true,
```

Comment:
@DenyHF Thank you for adding support to renderStart API. 
When I tested your example locally, I couldn't find the `render-start` msg to be sent. Could you please double check the implementation? Thank you.

Reply:
@zhouyx Thanks!
This was a problem on our side. We have already fixed it.

Reply:
Thank you @DenyHF 
I got this error when I test locally
![image](https://user-images.githubusercontent.com/5419904/60910722-65b7f880-a236-11e9-9d32-43dde89b532e.png)
Any ideas?

Reply:
Thank!
Fixed!

Reply:
Thank you @DenyHF, verified this is fixed.